paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewAction.kt

```diff
@@ -28,9 +30,12 @@ internal sealed class CustomerSheetViewAction {
         val mandateText: String?,
         val showAbovePrimaryButton: Boolean,
     ) : CustomerSheetViewAction()
-    class OnUSBankAccountRetrieved(
+    class OnConfirmUSBankAccount(
         val usBankAccount: PaymentSelection.New.USBankAccount,
     ) : CustomerSheetViewAction()
+    class OnCollectBankAccountResult(
+        val bankAccountResult: CollectBankAccountResultInternal,
+    ) : CustomerSheetViewAction()
```

Comment:
I’m not feeling great about these things being a view actions, as they are not originating from the view. I’m ok with continuing here, but we should be more rigid in the future.